components/search/Search.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import SearchDesktop from './SearchDesktop'
+import SearchMobile from './SearchMobile'
+import { useMediaQuery } from '@material-ui/core'
+import { useTheme } from '@material-ui/core/styles'
+import Spacer from 'react-storefront/Spacer'
+
+function Search() {
+  const theme = useTheme()
```

Comment:
Any specific reason to use this, instead of the `<Hidden>` that was used previously? Just wondering.

Reply:
As I understood with Hidden it will still render the component. So in this case it rendered for example two SearchProviders and when I did fetch it fetched twice.

Reply:
I think the default implementation of `<Hidden>` (which uses `implementation="js"`) returns `null` for hidden components. But it doesn't really matter, as this implementation is perfectly fine too. 😄

Reply:
Maybe for react components, but because SearchProvider is not react component it was not working? In any case it was doing 2 times SearchProvider with Hidden.